app/src/main/java/com/asfoundation/wallet/ui/iab/IabPresenter.kt

```diff
@@ -68,6 +69,13 @@ class IabPresenter(
     )
   }
 
+  private fun handleTryAgain() {
+    disposable.add(view.errorTryAgain()
+      .doOnNext { view.close(Bundle()) }
```

Comment:
When the user clicks "try again", it might be better to navigate back to the payment method screen instead of closing the payment flow here.